aries_vcx/src/protocols/connection/invitee/mod.rs

```diff
@@ -240,3 +243,60 @@ where
         self.state.bootstrap_did_doc()
     }
 }
+
+pub async fn any_invitation_into_did_doc(
```

Comment:
I don't think this function belongs here

Reply:
moved to `aries_vcx/src/protocols/connection/invitee/mod.rs`